app/controllers/api/irs_attempts_api_controller.rb

```diff
@@ -23,7 +23,8 @@ def create
 
         headers['X-Payload-Key'] = Base64.strict_encode64(result.encrypted_key)
         headers['X-Payload-IV'] = Base64.strict_encode64(result.iv)
```

Comment:
@n1zyy, I see that you restored the `X-Payload-` headers - did you decide that we needed them?

Reply:
yeah we need them